test/HardwareObjectsMockup.xml/mxcube-web/server.yaml

```diff
@@ -14,3 +14,8 @@ mxcube:
 
   usermanager:
     class: UserManager
+    inhouse_is_staff: True
+    users:
+      -
+        username: opid0
+        role: staff
```

Comment:
Just an example of assigning a role to a user directly from the UserManager configuration (not used yet)